dallasnews/dmn-vendor-wrapper
# Merge in dependabot version changes

## Description

Dependabot generated a number of PRs urging us to update package numbers. All of those package updates were rolled into this maintenance branch. Will all of those updates in place, we can test this branch and merge all changes at once.

## Testing

- Clone this repo if you have not already done so
- Checkout this branch
    - `checkout maintenance/dependabot`
- Install all packages
    - `npm i`
- Build Code
    - `npm run build`
- Run Server
    - npm run start
- Open a browser tab and navigate to ...
    - http://localhost:8080/
- You should see the page below

![Screen Shot 2023-02-02 at 4 15 45 PM](https://user-images.githubusercontent.com/5252624/216461701-155d71c8-13ca-49b6-b572-c18c29d872d9.png)

- Scroll the bottom of the page and click on the jobs link under the "ADVERTISE WITH US" section.
- This should take the user to https://dfwishiring.dallasnews.com/

![Screen Shot 2023-02-02 at 4 15 53 PM](https://user-images.githubusercontent.com/5252624/216461772-af8f9a0b-58ca-405c-b298-6ef58c348b82.png)
